utilities/cloudharness_utilities/deployment-configuration/helm/templates/auto-database.yaml

```diff
@@ -110,6 +110,121 @@ spec:
         persistentVolumeClaim:
           claimName: {{ .app.harness.database.name | quote }}
 ---
+{{ if eq .app.harness.database.type "postgres" }}
```

Comment:
Would be nice to have this split in different files: one file per database/backup:
like auto-database-postgres.yaml, auto-database-postgres-backup.yaml